Sources/GithubAPI.swift

```diff
@@ -266,4 +432,16 @@ extension GithubAPI {
       LogFile.error("error: \(error) desc: \(error.localizedDescription)")
     }
   }
+
+  func paginate(response: CURLResponse) -> String? {
+    if let links = response.get(HTTPResponseHeader.Name.custom(name: "Link")),
+      let nextLink = links.components(separatedBy: ",").filter({ $0.contains("rel=\"next\"") }).first,
+      let nextUrlAsString = nextLink.components(separatedBy: ";").first?
+        .trimmingCharacters(in: .init(charactersIn: " "))
```

Comment:
Maybe consider using .replaceOccurances(of: " ", with: "") just stylistic.

Reply:
`trimmingCharacters` removes the spaces from the beginning and end of a String whilst `replaceOccurances` removes all spaces from a string.

Reply:
LGTM